### ISSUE TITLE
"Spellcheck only comments" hangs the extension

### ISSUE THREAD

**Issue by seba-eng** [streetsidesoftware/vscode-spell-checker#368]
Using the regex from #107 in settings.json to check only comments in python hangs the extension for me in a 300 lines file with many docstrings.

**Comment by seba-eng** [comment]
I don't know if that's the source for the hanging, but discovered one problem:
`""" errror """` is detected as unknown word. But this:
```
""" errror
"""
```
is a valid commend in python but not detected.

relevant part of settings.json looks like this:
```
"cSpell.languageSettings": [
        {
            "languageId": "python",
            "includeRegExpList": [
                "/#.*/",
                "/\"\"\"(.*?\\n?)+?\"\"\"/g",
                "/'''(.*?\\n?)+?'''/g",
                "string"
            ]
        }
    ],
```

**Comment by Jason3S** [comment]
@seba-eng the original RegExp from #107 is incorrect and will cause the extension to hang. Please see discussion in #431.
I have updated #107 with the correct RegExp.
